cookiecutter/find.py

```diff
@@ -22,7 +22,7 @@ def find_template(repo_dir):
 
     project_template = None
     for item in repo_dir_contents:
-        if 'cookiecutter' in item and '{{' in item and '}}' in item:
+        if ('cookiecutter' in item or 'cc' in item) and '{{' in item and '}}' in item:
```

Comment:
Maybe better to combine it to shorter form? 
Hard to read so lot of in item

Reply:
Do you mind if we introduce regex here? -> `re.search(r'{{.*(cookiecutter|cc).*}}', item)`